### PR TITLE
Add setup notes

### PR DESCRIPTION
**Pull request by schpet**
- Add db setup step to readme
- generate a prisma migration based on the schema

### CODE REVIEW

**Comment by gadogado** [comment]
hey thanks for this!  after looking through the current migrations I was pretty puzzled on how production ( and my local ) already have these changes .... but I must have pushed the schema w/ the prisma cli in both environments at some time.

I'll pull this in soon 👍

**Comment by schpet** [comment]
@gadogado yeah i was thinking, since fly doesn't use git, maybe you _had_ that migration and `flyctl deploy`'d it, but never committed it to git and accidentally deleted it? i like how fly's deploys do not use git but definitely run into surprises about what gets deployed sometimes.

File: README.md [hunk 1]
```diff
@@ -9,11 +9,12 @@
...
-5.  Build the app: ```npm run build```
-6.  Run the process that grabs, normalizes, and persists the ycombinator job post items and stories: ```npm run manual-get-latest``` NOTE:  You can optionally increase how effective the `Promise.all` is when fetching firebase items by changing (increase) the `.env` value `CONCURRENCY_LIMIT=`. This largely depends on the architecture of your local machine.  
-7.  Run the server: ```npm run dev``` and visit `http://localhost:3000`
+3. hn-hired uses the [Prisma](https://www.prisma.io/docs/concepts/database-connectors/postgresql) ORM with Postgres.  You'll need to create a new local db and update the `.env` value for `DATABASE_URL=` with the respective db url, e.g., `createdb hnhired && postgresql://postgres:postgres@localhost:5432/hnhired` `(postgresql://USER:PASSWORD@HOST:PORT/DATABASE)` 
+4.  run `npm run setup` to setup the database
```
Comment (gadogado) [inline README.md:13]: can ditch this duplicate step ( was in 4 )

**Comment by gadogado** [comment]
> @gadogado yeah i was thinking, since fly doesn't use git, maybe you _had_ that migration and `flyctl deploy`'d it, but never committed it to git and accidentally deleted it? i like how fly's deploys do not use git but definitely run into surprises about what gets deployed sometimes.

ya totally.  also nice to have the power of prisma db push but probably for quick local prototyping only ha 😄

File: README.md (hunk 1 above)
Comment (schpet) [reply README.md]: ah lol, yeah too late for me to read all that

**Pull request merged**